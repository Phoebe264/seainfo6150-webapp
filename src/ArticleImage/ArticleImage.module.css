.img{

 width: 220px;
 height: 150px;

 border-style: solid;
 border-width: 0.7rem;
 border-radius: 50%;
 border-color: rgb(145, 8, 45);

 margin-bottom: 1.5rem;

 }