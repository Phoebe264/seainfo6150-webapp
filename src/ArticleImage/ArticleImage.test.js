import React from "react";
import ArticleImage from './ArticleImage'

it("renders correctly", () => {
  const { container } = render(<ArticleImage url={"https://cdn.cnn.com/cnnnext/dam/assets/170124150859-beautiful-india-kerala-super"} title={"You can take a pie on a plane"} />);
  expect(container).toMatchSnapshot();
});