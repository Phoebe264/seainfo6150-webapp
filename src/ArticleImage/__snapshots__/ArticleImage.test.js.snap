// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <img
    alt="You can take a pie on a plane"
    class="image"
    src="https://cdn.cnn.com/cnnnext/dam/assets/170124150859-beautiful-india-kerala-super"
  />
</div>
`;
